hub/api/objectview.py

```diff
@@ -0,0 +1,373 @@
+from hub.api.datasetview import DatasetView
+from hub.schema import Sequence, Tensor, SchemaDict, Primitive
+from hub.api.dataset_utils import slice_extract_info, slice_split, str_to_int
+
+# from hub.exceptions import NoneValueException
+import collections.abc as abc
+import hub.api as api
+
+
+class ObjectView:
+    def __init__(
+        self,
+        dataset,
+        subpath=None,
+        slice_list=None,
+        nums=[],
+        offsets=[],
+        squeeze_dims=[],
+        inner_schema_obj=None,
+        new=True,
+    ):
```

Comment:
Please add documentation for the arguments.

Reply:
Done